### ISSUE TITLE
popravitobase: Report a problem (2nd step) missing elements and font issue

### ISSUE THREAD

**Issue by schlos** [codeforcroatia/popravi-to#51]
**Describe the bug**

Report a problem screen (2nd step) is missing elements and font issue:
- missing progress bar on top
- title/text font is wrong
- category and subcategory dropdown is too narrow
- category and subcategory font color is wrong
- 'Summarize the problem', 'Photos' and 'Explain what's wrong' are missing (i) icon
- 'e.g.' text under 'Explain what's wrong' should be located under the text box
- 'e.g.' text under 'Explain what's wrong' text box should have smaller font size
- 'Continue' button on the bottom of page is missing a symbol

**To Reproduce**

**Expected behavior**

During dev, we agreed that 2 Figma screens will be created on a single screen ([screen 1](https://www.figma.com/proto/OrvZfg5BsfdOzeL7nhRpEI/Popravi.to?node-id=543%3A6090&scaling=min-zoom&page-id=543%3A5650&hide-ui=1), [screen 2](https://www.figma.com/proto/OrvZfg5BsfdOzeL7nhRpEI/Popravi.to?node-id=543%3A6144&scaling=min-zoom&page-id=543%3A5650&hide-ui=1)) as application is not yet added support for multi-screen wizard in this step. See marked on screenshots below what needs fixing:
- add progress bar on top
- fix title/text font
- fix category and subcategory dropdown height
- fix category and subcategory font color
- add (i) icon next to 'Summarize the problem', 'Photos' and 'Explain what's wrong' - mind that click on this icon should open a new modal, each with different texts (see Figma)
- move 'e.g.' text under 'Explain what's wrong' to proper location under the text box
- fix font size for 'e.g.' text under 'Explain what's wrong' text box
- add arrow symbol on 'Continue' button on the bottom of page

**Screenshots**

Current problems (on new web "popravitobase" cobrand):
https://snippyly.com/img/?q=uDDIm66KIUu9u3iGzaXv (&EDIT=4rfDu9SeRnarQmqSKX0U)

Expected (from Figma design) - 2 design screen are agreed to have it on a single step screen:
https://snippyly.com/img/?q=i0H0XHjmjm9JoQwoCEf7 (&EDIT=wVPouXAzMpkYrUcYvv6K)
https://snippyly.com/img/?q=HTif7BChWqZGF3cte1Li (&EDIT=WBaCBY2ilyGokpk8BeoU)

Expected (from Figma design) - modal for (i) icon:
https://snippyly.com/img/?q=vd0Kif9N1PE3hkZ4eRf1 (&EDIT=Too8jGHJVFb80FlSSOGi)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]

**Smartphone (please complete the following information):**

**Additional context**

**Comment by schlos** [comment]
Kostya feedback:
To fix [it] I need links to where can I create another page thats not connected and how to add jquery to fixmystreet

**Comment by schlos** [comment]
Let's break it down:
1) _"To fix [it] I need links to where can I create another page thats not connected"_ 
- Why is this needed? 
- You mean how to create another *.html page? 
- What you mean by "not connected"? 
- And how do you plan to use this new page?

You create new page like this:
`sudo -u fms touch new_page.html`

And here is instruction how to include content of that page to another page:
https://fixmystreet.org/customising/templates/
Usually it's included like this
`[% INCLUDE 'new_page.html' %]`

2) _"how to add jquery to fixmystreet"_

It's same as adding CSS file, please see instructions:
https://fixmystreet.org/customising/css/

Basically, you need to add your JS files to `/web/cobrands/<name_of_your_cobrand>/`

**Comment by schlos** [comment]
Not fixed still. See comments on screenshots:

https://snippyly.com/img/?q=f6lH994reiBVC03H2z3j &EDIT=L87fG1K2kEIdGEvd46ax
https://snippyly.com/img/?q=Ib7HUCFzw1STqyPb2a2E &EDIT=eIVkXHTNwxaZDChBHeCu
